<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>TcpSocketServer</class>
  <widget class="QWidget" name="TcpSocketServer">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>441</width>
        <height>404</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>TcpSocketServer</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
            <string>接收显示</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
              <number>12</number>
            </property>
            <property name="topMargin">
              <number>12</number>
            </property>
            <property name="rightMargin">
              <number>12</number>
            </property>
            <property name="bottomMargin">
              <number>12</number>
            </property>
            <item row="0" column="0">
              <widget class="QTextBrowser" name="textBrowser" />
            </item>
          </layout>
        </widget>
      </item>
      <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
            <string>服务端配置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
              <number>12</number>
            </property>
            <property name="topMargin">
              <number>12</number>
            </property>
            <property name="rightMargin">
              <number>12</number>
            </property>
            <property name="bottomMargin">
              <number>12</number>
            </property>
            <item row="0" column="0">
              <widget class="QLabel" name="label">
                <property name="text">
                  <string>监听地址</string>
                </property>
              </widget>
            </item>
            <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_ServerAddress">
                <property name="text">
                  <string>127.0.0.1</string>
                </property>
              </widget>
            </item>
            <item row="0" column="2">
              <widget class="QLabel" name="label_2">
                <property name="text">
                  <string>监听端口</string>
                </property>
              </widget>
            </item>
            <item row="0" column="3">
              <widget class="QSpinBox" name="spinBox_ServerPort">
                <property name="maximum">
                  <number>65535</number>
                </property>
                <property name="value">
                  <number>4800</number>
                </property>
              </widget>
            </item>
            <item row="0" column="4">
              <widget class="QPushButton" name="pushButton_ServerControl">
                <property name="text">
                  <string>开始</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_ManulControl">
          <property name="title">
            <string>手动控制</string>
          </property>
          <property name="checkable">
            <bool>true</bool>
          </property>
          <property name="checked">
            <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
              <number>12</number>
            </property>
            <property name="topMargin">
              <number>12</number>
            </property>
            <property name="rightMargin">
              <number>12</number>
            </property>
            <property name="bottomMargin">
              <number>12</number>
            </property>
            <item>
              <widget class="QComboBox" name="comboBox_ClientConnected">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="lineEdit_MessageContent" />
            </item>
            <item>
              <widget class="QPushButton" name="pushButton_SendMessage">
                <property name="text">
                  <string>发送</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <resources />
  <connections />
</ui>
